File: java/src/test/java/mx/iteso/ut/TortillaHarinaTest.java
package mx.iteso.ut;

import org.junit.Test;
import org.junit.Before;
import static org.junit.Assert.*;

public class TortillaHarinaTest {
    TortillaHarina tortillaHarina;

    @Before
    public void setup(){
        tortillaHarina = new TortillaHarina();
    }

    @Test
    public void TestCurrentTemperature(){
        tortillaHarina.setCurrentTemperature(15);
        assertEquals(15,tortillaHarina.getCurrentTemperature());
    }

    @Test
    public void TestTrueToasted(){
        tortillaHarina.toast(true);
        assertTrue(tortillaHarina.isToasted());
    }

    @Test
    public void TestFalseToasted(){
        tortillaHarina.toast(false);
        assertFalse(tortillaHarina.isToasted());
    }

    @Test
    public void TestToasting(){
        assertEquals(10, tortillaHarina.getToastTemperature());
    }
}

File: python/src/unittest/python/TortillaHarinaTest.py
from TortillaHarina import *
from mockito import when, mock, verify,assertEquals,assertFalse,assertTrue
import unittest
class TortillaHarinaTest(unittest.TestCase):
    def __init__(self):
        self.tortillaHarina = TortillaHarina()

    def testCurrentTemperature(self):
        self.tortillaHarina.setCurrentTemperature(11)
        assertEquals(11,self.tortillaHarina.getCurrentTemperature())

    def testFalseToast(self):
        self.tortillaHarina.toast(False)
        assertFalse(self.tortillaHarina.isToasted())

    def testTrueToast(self):
        self.tortillaHarina.toast(True)
        assertTrue(self.tortillaHarina.isToasted())

    def testToasting(self):
        assertEquals(10,self.tortillaHarina.getToastTemperature())